action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
   version:
     description: Version of Enos CLI to install
     required: false
-    default: '0.0.16'
+    default: '0.0.18'
```

Comment:
0.0.16 the latest version of the Enos CLI https://github.com/hashicorp/enos/releases/tag/v0.0.16